docs/reference/server-config.md

```diff
@@ -359,6 +361,29 @@ Example:
 	</tr>
 </table>
 
+## Hot logging level reload
+We don't support to completely reload the whole configuration yet at present. What we support for now is:
+- increase logging level by signaling `SIGUSR1`
+- decrease logging level by signaling `SIGUSR2`
+
+Example:
+
+To increase logging level
+```
+$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
```

Comment:
Thanks :)

On Monday, June 13, 2016, HuKeping <notifications@github.com> wrote:

> In docs/reference/server-config.md
> <https://github.com/docker/notary/pull/748#discussion_r66908468>:
>
> > @@ -359,6 +361,29 @@ Example:
> >  	</tr>
> >  </table>
> >
> > +## Hot logging level reload
> > +We don't support to completely reload the whole configuration yet at present. What we support for now is:
> > +- increase logging level by signaling `SIGUSR1`
> > +- decrease logging level by signaling `SIGUSR2`
> > +
> > +Example:
> > +
> > +To increase logging level
> > +```
> > +$ kill -s SIGUSR1 `ps aux | grep "notary-server -config" | grep -v "grep" | awk '{print $2}'`
>
> OK, I'll update it later.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/notary/pull/748/files/3d60810a1c0e00af7c55324ade3032fc07aa1b2a#r66908468>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AANn4Axf7p4eoevLSpWJKeZRMnHrEvtFks5qLjOEgaJpZM4Iel-p>
> .
>